packages/tasks/src/tasks/zero-shot-object-detection/about.md

```diff
@@ -0,0 +1,38 @@
+## About the task
+
+Zero-shot object detection is a computer vision task to detect objects and its class in images, without any prior training or knowledge of the classes.
+
+Zero shot object detection works by transferring knowledge learnt during training of one model, to classify novel classes that was not present in the training data.
```

Comment:
We can also append it to above text and directly write to `summary` instead

Reply:
Should I remove both parts in the `about.md` file and instead write them in the `summary` of `data.ts` or remove keep them and add them to the `data.ts` file?